develop-guide/writeblitz/writeblitz.rst

```diff
@@ -1371,6 +1482,27 @@ There are a lot of use cases for looping with various features. Examples can be
         - sleep:
             sleep_time: "%VARIABLES{range_name}"
 
+.. code-block:: YAML
+
+    # running a loop with loop_until: passed
+    # The iteration stops the second the last iteration is equal to passed.
```

Comment:
If you put True,  will it take it as a bool or a string/

Reply:
True goes as bool